documentation/getting_started.md

```diff
@@ -730,3 +729,28 @@ To change the screen to the demonstrate the scenario, 1) open the modal navigati
 After changing the screen, you can see the **Camera Preview** and the real-time object classification result above the preview box as shown below.
 
 ![Android example](img/A-34-arm64_VisionEx0.png)
+
+#### Handling ML Computation Delegation Requests from the Android Components in the Other package
+
+The MLAgent can handle the ML Computation from the other package. The following code should be added to the other package's `AndroidManifest`
+for [package visibility](https://developer.android.com/training/package-visibility/declaring).
+
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <queries>
+        <package android:name="ai.nnstreamer.ml.inference.offloading" />
+        <package android:name="ai.nnstreamer.ml.inference.offloading.MainService" />
+    </queries>
+    ...
+</manifest>
+```
+
+We are providing a simple example application that demonstrates this scenario. This example sends an input prompt to the MLAgent
+and retrieves the generated output text using the llama2 model.
+
+To receive auto-generated text by llama2 model, 1) enter the beginning of the sentence and 2) press the `Run llama2` button.
+
+![Android example](img/A-34-arm64_LlamaEx0.jpeg)
+
+If you want to build this example, you can follow [the guide](https://github.com/nnstreamer/nnstreamer-example/tree/doc/android/kotlin_app#android-application-examples-using-mlops-solution).
```

Comment:
This guide is under revewing
https://github.com/nnstreamer/nnstreamer-example/pull/362